scripts/deploy-script.js

```diff
@@ -20,6 +20,16 @@ async function main() {
   await lanVar.deployed();
 
   console.log("LanVar deployed to:", lanVar.address);
+
```

Comment:
we should separate to 2 deploy scripts just in case we just need to update 1 of 2 contracts at a time